### PR TITLE
[Theme] A very minimalistic theme

### PR DESCRIPTION
**Pull request by Jikstra**
Screenshot_20210118_212508

# ToDo

- [x] Fix context menu in map component

### CODE REVIEW

File: bin/build-shared-version-info.js [hunk 1]
```diff
@@ -2,14 +2,25 @@ const { spawnSync } = require('child_process')
 const fs = require('fs-extra')
```
Comment (Simon-Laux) [inline bin/build-shared-version-info.js:2]: unrelated file to this pr

**Comment by Jikstra** [comment]
I don't want this pr to get older and test this out in actual usage. We can still revert parts of this or tweak things.

File: src/main/deltachat/contacts.ts [hunk 2]
```diff
@@ -78,7 +78,13 @@ export default class DCContacts extends SplitOut {
...
 
   getContact(contactId: number) {
-    return this._dc.getContact(contactId).toJson()
+    const contact = this._dc.getContact(contactId).toJson()
+    if (contactId === C.DC_CONTACT_ID_SELF) {
+      contact.firstName = contact.name = contact.displayName = this._dc.getConfig(
+        'displayname'
+      )
+    }
+    return contact
```
Comment (Simon-Laux) [inline src/main/deltachat/contacts.ts:87]: Could you make an issue on core to discuss moving this hack to core.
